src/main/kotlin/ecommerce/config/WebMvcConfiguration.kt

```diff
@@ -27,4 +30,16 @@ class WebMvcConfiguration(
             .addPathPatterns("/api/**")
             .excludePathPatterns("/api/members/login", "/api/members/register")
     }
+
+    override fun addCorsMappings(registry: CorsRegistry) {
+        registry.addMapping("/api/**")
+            .allowedOrigins("*")
```

Comment:
This is just a TMI that I think would be great for us to cover, rather than actually changing the logic.

For now, `allowedOrigins("*")` works fine, but if we're making APIs that are only for our team or product, we need to narrow down the allowed origins to expected client information. If we're making public API, then the situation is different, but in most of the times, we make APIs that are used in only certain places. We'll get to that situation at some point I guess (it might even be in team mission 🙂 )